src/public/audits-table.vue

```diff
@@ -256,6 +379,133 @@ export default {
+            if (labels = this.actionLabelFromAuditLabel(audit)) {
+              return labels
+            }
+
+            // Case 2. Action Label can be fully defined by the Audit's Event Type alone
+            if (labels = this.actionLabelFromEventType(audit)) {
+              return labels
+            }
+
+            // Case 3. Action Labels needs to be defined by a combination of several data (creating potentially a multi-line label)
+            if (labels = this.actionLabelFromAuditChanges(audit)) {
+              return labels
+            }
+        },
+
+        actionLabelFromAuditLabel(audit) {
+          const searchKey = 'label'
+          const result = this.text_configs.find(item => item.match[searchKey] === audit.label)
+          let interpreted
+          if (result) {
+            interpreted = {
+              ...result,
+              text: this.evaluateExpression(result.text, {audit})
+            }
+
+            return [interpreted]
+          }
+        },
+
+        actionLabelFromEventType(audit) {
+          const searchKey = 'event_type'
+          let interpreted
+          const result = this.text_configs.find(item => item.match[searchKey] === audit.event_type)
+          if (result) {
+            interpreted = {
+              ...result,
+              text: this.evaluateExpression(result.text, {audit})
+            }
+
+            return [interpreted]
+          }
+        },
+
+
+        actionLabelFromAuditChanges(audit) {
+          const actions = []
+
+          for (const [attribute, change] of Object.entries(audit.changes)) {
+            const label = this.actionLabelFromAuditChange(attribute, change, audit)
+
+            label && actions.push(label)
+          }
+          if (!actions.length == 0) {
+            return actions
+          }
+        },
+
+        actionLabelFromAuditChange(attribute, change, audit) {
+          let searchKey = 'attribute'
+          let result = this.text_configs.find(item => item.match[searchKey] === attribute)
+          let interpreted
+          if (result) {
+            interpreted = {
+              ...result,
+              text: this.evaluateExpression(result.text, {audit, change, attribute})
+            }
+
+            return interpreted
+          } else {
+            searchKey = 'event_type_generic'
+            result = this.text_configs.find(item => item.match[searchKey] === audit.event_type)
+            if (result) {
+              interpreted = {
+                ...result,
+                text: this.evaluateExpression(result.text, {audit, change, attribute})
+              }
+
+              return interpreted
+            }
+          }
+        },
+
+        evaluateExpression(expr, context) {
+            return new Function('context', `with (context) {return ${expr};}`)(context)
+        },
+
+        onUserFilterChange(id) {
+            this.filter.user_id=id
```

Comment:
Can you please add a space around the equals sign (here and the 2 others below) for consistency with the general format elsewhere?